.travis.yml

```diff
@@ -49,6 +50,15 @@ script:
     --filename posterior_sites
     --outfile tests/unit/combine_results/combined_MotEvo_results.tsv
   - md5sum --check tests/unit/combine_results/expected_output.md5
+  - conda activate plot_heatmap_of_MotEvo_results
```

Comment:
After you build all ther other conda v-envs (previous message), could you please `activate` them before each respective unit test as well?